fold.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+
+int main()
+{
+    int rows = 11;
+    int columns = 11;
+
+    for (int j{1}; j <= rows; ++j)
```

Comment:
Please don't use `{}` syntax to set variables. Normal `=` assignment is far clearer